src/game/client/draw/w3doverlordaircraftdraw.cpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief W3D Overlord Aircraft Draw Module
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "w3doverlordaircraftdraw.h"
+#include "drawable.h"
+#include "object.h"
+#include "opencontain.h"
+
+void W3DOverlordAircraftDrawModuleData::Build_Field_Parse(MultiIniFieldParse &p)
+{
+    // clang-format off
+    static const FieldParse dataFieldParse[] = {
+        { nullptr, nullptr, nullptr, 0 },
+    };
+    // clang-format on
+
+    W3DModelDrawModuleData::Build_Field_Parse(p);
+    p.Add(dataFieldParse, 0);
+}
+
+NameKeyType W3DOverlordAircraftDraw::Get_Module_Name_Key() const
+{
+    static const NameKeyType nk = g_theNameKeyGenerator->Name_To_Key("W3DOverlordAircraftDraw");
+    return nk;
+}
+
+void W3DOverlordAircraftDraw::Do_Draw_Module(const Matrix3D *transform)
+{
+    W3DModelDraw::Do_Draw_Module(transform);
+    Object *object = Get_Drawable()->Get_Object();
+
+    if (object != nullptr && object->Get_Contain() != nullptr) {
+        const Object *rider = object->Get_Contain()->Friend_Get_Rider();
+
+        if (rider != nullptr) {
+            captainslog_dbgassert(rider->Get_Drawable() != nullptr, "OverlordAircraftDraw finds no rider's drawable");

```

Comment:
Please simplify this code to improve readability.
```c++
Drawable *drawable = rider->Get_Drawable();
if (drawable != nullptr) {
...
} else {
    captainslog_dbgassert(0, "OverlordAircraftDraw finds no rider's drawable");
}
```

Reply:
fixed.